Libraries/Utilities/Platform.ios.js

```diff
@@ -12,8 +12,21 @@
 
 'use strict';
 
+var invariant = require('invariant');
+
 var Platform = {
   OS: 'ios',
+  select: (obj: Object) => {
+    invariant(
```

Comment:
Remove this one as well. This is going to be a hot path and the error isn't very useful.